src/lib/client.ml

```diff
@@ -185,6 +186,12 @@ module Http_client = struct
     in
     call_up_msg (`Get_target_ids query)
 
+  let call_process_holder t message =
+    call_json t ~path:"/api" ~meta_meth:(`Post_message (`Process message))
+    >>= fun down ->
+    filter_down_message down ~loc:`Process_holder
+      ~f:(function `Process p -> Some p | other -> None)
+
 
 end
```

Comment:
Not a fan of those comments; it's like encoding in the source something that should be a text-editor feature (like using `%` http://vim.wikia.com/wiki/Moving_to_matching_braces).